websrc/source/documents/networking/services.md

```diff
@@ -17,16 +17,95 @@ Services offer:
+
+````
+	[vagrant@netplugin-node1 ~]$ netctl service ls
+	ServiceName    Tenant    Network      Selectors
+	---------      --------  -------      -------
+	svc-0-default  default   svc-net-1-0  [key0=value1 key1=value2 key2=value3 key3=value4]
+
+	[vagrant@netplugin-node1 ~]$ netctl service inspect svc-0-default
+	----
+	----
+	        "labels": "map[key0:value1 key1:value2 key2:value3 key3:value4]",
+        "macAddress": "02:02:0a:00:00:08",
+        "network": "net0-1.default",
+        "serviceName": "svc-0-default"
+      }
+    ],
+    "serviceVip": "30.1.0.1" 
+  }
+````
+
+4\. Create four service containers in network net0-2
+	
+    docker run -itd --name=net0-2-srv0-0-0 --net=net0-2  contiv/alpine  sleep 600m
+    docker run -itd --name=net0-2-srv1-1-1 --net=net0-2  contiv/alpine  sleep 600m
+    docker run -itd --name=net0-2-srv2-2-2 --net=net0-2  contiv/alpine  sleep 600m
+    docker run -itd --name=net0-2-srv3-0-3 --net=net0-2  contiv/alpine  sleep 600m
+
+5\. Similarly create  provider containers as service backend containers 
+	
+    docker run -itd --name=srv-default-net0-2-20-3 --net=net0-2  --label=key0=value1 --label=key1=value2 --label=key2=value3 --label=key3=value4  contiv/alpine sleep 600m
+    docker run -itd --name=srv-default-net0-2-21-1 --net=net0-2  --label=key0=value1 --label=key1=value2 --label=key2=value3 --label=key3=value4  contiv/alpine sleep 600m
+    docker run -itd --name=srv-default-net0-2-22-2 --net=net0-2  --label=key0=value1 --label=key1=value2 --label=key2=value3 --label=key3=value4  contiv/alpine sleep 600m
+    docker run -itd --name=srv-default-net0-2-23-0 --net=net0-2  --label=key0=value1 --label=key1=value2 --label=key2=value3 --label=key3=value4  contiv/alpine sleep 600m
+
+6\. Run Service  instances on provider containers(TCP server at ports 8080 and 7070) i.e Use netcat utility to run server 
+
+    docker exec -it srv-default-net0-2-20-3  nc -lk -p 7070 -e /bin/true &
+    docker exec -it srv-default-net0-2-21-1  nc -lk -p 7070 -e /bin/true &
+    docker exec -it srv-default-net0-2-22-2  nc -lk -p 7070 -e /bin/true &
+    docker exec -it srv-default-net0-2-23-0  nc -lk -p 7070 -e /bin/true &
+    docker exec -it srv-default-net0-2-20-3  nc -lk -p 8080 -e /bin/true &
+    docker exec -it srv-default-net0-2-21-1  nc -lk -p 8080 -e /bin/true &
+    docker exec -it srv-default-net0-2-22-2  nc -lk -p 8080 -e /bin/true &
+    docker exec -it srv-default-net0-2-23-0  nc -lk -p 8080 -e /bin/true & 
+
+7\. login into service conatiner and connect to service  i.e Use netcat utility to run client to connect server
+
+    docker exec -it net0-2-srv1-1-1  nc -z -n -v -w 1  30.2.0.1 80
+    docker exec -it net0-2-srv3-0-3  nc -z -n -v -w 1  30.2.0.1 80
+    docker exec -it net0-2-srv2-2-2  nc -z -n -v -w 1  30.2.0.1 80
+    docker exec -it net0-2-srv0-0-0  nc -z -n -v -w 1  30.2.0.1 80
+	
+8.\  As above operation result, client utiltiy  must be connected to the service 
```

Comment:
I'm not sure what this sentence is trying to say.  What is the "client utility"?  Why is there no text under this step?

"utiltiy" -> "utility"

Reply:
I think , you are right . this sentence  shouldn't be exist. docker exec operation itself explain the outcome.